client/components/comments/index.js

```diff
@@ -94,12 +94,21 @@ class Comments extends Component {
 const mapStateToProps = ({ comments, application: { commentable, showComments } }, { field }) => {
   const name = field.split('.').pop();
   let allComments = comments[field];
-  // backwards compatibility fix for some comments being saved without a prefix
+
+  // backwards compatibility fixes for some comments being saved without a prefix
   // merge comments saved with unprefixed name and full name
-  // whitelist the fields in legacy protocol animal questions as they are the only ones affected
-  if (name !== field && field.match(/^protocols\.[a-f0-9-]+\.item\./)) {
-    allComments = [].concat(comments[name]).concat(comments[field]).filter(Boolean);
+  if (name !== field) {
+    // whitelist the fields in legacy protocol animal questions
+    if (field.match(/^protocols\.[a-f0-9-]+\.item\./)) {
+      allComments = [].concat(comments[name]).concat(comments[field]).filter(Boolean);
+    }
+
+    // whitelist the fields nested under the fate section of protocols
+    if (field.match(/^protocols\.[a-f0-9-]+\.killed|killing-method|method-and-justification|continued-use-relevant-project/)) {
```

Comment:
Does this need a `$` at the end of the regex to avoid matching partial field names?

Reply:
It's already limited to just the protocol fields, but sure.

Reply:
```suggestion
    if (field.match(/^protocols\.[a-f0-9-]+\.(killed|killing-method|method-and-justification|continued-use-relevant-project)$/)) {
```

Reply:
I'm not 100% confident about regex structure and order-of-operations but I added `(...)` to make sure the `|` were explicitly scoped as well.